code_eval.py

```diff
@@ -89,16 +89,22 @@ def python3(cmd):
 		output = 'unknown error'
 	reply(cmd, output)
 
+def _no_back_ticks(str):
+	return str.strip('`').strip()
```

Comment:
what's the second `.strip()` for? I'm inclined to just get rid of it and then inline this function

Reply:
leading and trailing white-space @see https://docs.python.org/3/library/stdtypes.html#str.split

Reply:
sorry, I meant what is the purpose of stripping the leading and trailing whitespace? can you show me a test where it's necessary?